packages/cloud_firestore/cloud_firestore/example/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
```

Comment:
@Salakar I had to update Podfile in order to make the build work; as my local Flutter version (1.20.4) complained about it. Would you accept a separate PR that updates all the Podfiles in the repo? I don't like to _pollute_ this PR with such change.

Reply:
> Would you accept a separate PR that updates all the Podfiles in the repo

Yes please! Always gladly accept improvements to the examples, some of them are falling behind or get missed from Flutter updates (again working on CI/tooling to highlight these in future).

Reply:
Sorry to hijack this a a bit, I am also seeing the following error when running e2e tests:

```
6.33.0 - [Firebase/Core][I-COR000005] No app has been configured yet.
6.33.0 - [Firebase/Core][I-COR000008] The project's Bundle ID is inconsistent with either the Bundle ID in 'GoogleService-Info.plist', or the Bundle ID in the options if you are using a customized options. To ensure that everything can be configured correctly, you may need to make the Bundle IDs consistent. To continue with this plist file, you may
change your app's bundle identifier to 'io.flutter.plugins.firebase.firestore.example'. Or you can download a new configuration file that matches your bundle identifier from https://console.firebase.google.com/ and replace the current one.
flutter: Observatory listening on http://127.0.0.1:62603/IHNJ1bN5Q0o=/
```

Followed by:

```
6.33.0 - [Firebase/Firestore][I-FST000001] WriteStream (7fd48f00d538) Stream error: 'Unavailable: failed to connect to all addresses'
```

The tests will timeout after that.. is this related to example files not being updated?

Reply:
> The tests will timeout after that.. is this related to example files not being updated?

Will need to start up the Firestore emulator so it can connect locally;

```
cd .github/workflows/scripts
./start-firestore-emulator.sh
```

Needs an addition to the contribution guide really, sorry about that!